staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -2054,3 +2057,20 @@ type NetworkConfiguration struct {
 // +k8s:openapi-gen=true
 type GuestAgentPing struct {
 }
+
+// FlavorMatcher references a flavor that is used to fill fields in the VMI template.
+// +k8s:openapi-gen=true
+type FlavorMatcher struct {
```

Comment:
@davidvossel do you remember our discussion about references on the PVC Autoupdate story.
Do you think we should apply that here as well and instead of the `GloballyScoped` boolean to a proper GVK reference that defaults to either one value but also has the `Profile` field?

Reply:
Can you point me to the discussion, or summarize it here, so I know what you mean?

Reply:
This is the discussion: https://github.com/kubevirt/community/pull/99#discussion_r630234765
And more on the references topic from Kubernetes: https://docs.google.com/document/d/1N2MbLsO0xLCa4PZkEM8g3Skt7XKv8C49_6yW1gBiMic/edit#heading=h.m11bjbx2s877
https://github.com/kubernetes/community/pull/5748

Reply:
> Do you think we should apply that here as well and instead of the GloballyScoped boolean to a proper GVK reference that defaults to either one value but also has the Profile field?

what would be the value in doing this? I can't think of any